std/range/package.d

```diff
@@ -3515,8 +3515,26 @@ private alias lengthType(R) = typeof(R.init.length.init);
    offers random access iff all ranges offer random access, and also
    offers mutation and swapping if all ranges offer it. Due to this, $(D
    Zip) is extremely powerful because it allows manipulating several
-   ranges in lockstep. For example, the following code sorts two arrays
-   in parallel:
+   ranges in lockstep.
+   For example, the following code sorts two arrays in parallel:
+
+   -------
+   import std.algorithm : sort;
+
+   int[] a = [ 1, 2, 3 ];
+   string[] b = [ "a", "b", "c" ];
+
+   sort!((c, d) => c[0] > d[0])(zip(a, b));
+
+   assert(a == [ 3, 2, 1 ]);
+   assert(b == [ "c", "b", "a" ]);
+   -------
+
+   See_Also: $(LREF lockstep)
+
+      $(LREF lockstep) is similar to $(D zip), but it bundles its values
+      and returns a range. $(LREF lockstep) is useful for
```

Comment:
I updated the comment strings, please have a look again. This time I hope I didn't mess up the description ;-)